data/mods/blazemod/blaze_autoweapons.json

```diff
@@ -41,7 +41,7 @@
     "price": 280000,
     "material": ["steel", "plastic"],
 	"ammo_effects": ["BOUNCE", "LIGHTNING", "MOUNTED_GUN"],
-    "flags": ["NO_AMMO", "NEVER_JAMS",  "NO_UNLOAD", "USE_UPS_200"],
+    "flags": ["NEVER_JAMS",  "NO_UNLOAD", "USE_UPS_200"],
```

Comment:
The "USE_UPS_*" flags don't do anything anymore. The used UPS charges are defined directly as a member:
```JSON
{
    "id": "ateslagun",
    "ups_charges": 200,
```

Hmm, seems I haven't properly updated that in all the mods when I changed it, I apologize.